clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -950,6 +950,10 @@ public void assign(Collection<TopicPartition> partitions) {
      *             topics or to the configured groupId
      * @throws org.apache.kafka.common.KafkaException for any other unrecoverable errors (e.g. invalid groupId or
      *             session timeout, errors deserializing key/value pairs, or any new error cases in future versions)
+     * @throws java.lang.IllegalArgumentException.IllegalArgumentException if a negative value is provided for the
+     *             timeout parameter
+     * @throws java.lang.IllegalStateException.IllegalStateException is the consumer is not subscribed to any topics or
```

Comment:
Sorry, my bad. It seems the constructor's fully qualified name was copied and pasted there. Will also update the typo. Thanks for catching those.